script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -376,27 +375,28 @@ The man reveals his true colors in a voice that could be described as respectful
 [sha:6bba3bd1f03fd2cd096daba4a6eecb3778beb02d]{
 -- Page 542, Offset 10054.
 -- 人間を嫌い、恐れ、避けようと絶海を漂流し、城を構えた変わりもの。
-A freak who hated and feared humans, who drifted across distant seas and built a castle to avoid them.
+There was an outsider who so despised and feared humans that in order to avoid them, they set themselves adrift among the distant oceans, holding there a castle.
```

Comment:
As in their castle (literal/metaphorical) was literally on a boat?

Reply:
We don't know of the details, but Zaria's "home" was instead "the distant oceans", probably the ice floating around up in the northern part of the Earth.